parlai/tasks/coqa/build.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import parlai.core.build_data as build_data
+import os
+import json
+
+
+def make_parlai_format(outpath, dtype, data):
+    print('building parlai:' + dtype)
+    fout = open(os.path.join(outpath, dtype + '.txt'), 'w')
+    for each in data:
+        s = ""
+        for question, ans in zip(each['questions'], each['answers']):
+            s += "text:"
+            if question['turn_id'] == 1:
+                s += each['story'] + '\n'
+            s += question['input_text'] + '\t\t'
+            s += "labels:" + ans['input_text']
```

Comment:
you need to escape the labels with `ans['input_text'].replace("|", " __PIPE__ ")` due to '|' having special semantics in the file format.

Reply:
Could you elaborate a bit on what that is doing?